session-name-1/README.md

```diff
@@ -0,0 +1,49 @@
+# Session Name
+
+## Introduction
+
+Objectives and rationale for the current session
+
+## Reminders and Prerequisites
+
+Information required for this section
+
+Commands / snippets that should be known, useful to copy-paste throughout the practical session
+
+## Section 1
+
+Content split in sections, according to session specifics
+
+Demos will be part of the session presentation and will be referenced (snippets, images, links) in the content
+
+## Section 2
+
+Content split in sections, according to session specifics
+
+Demos will be part of the session presentation and will be referenced (snippets, images, links) in the content
+
+## Summary
+
+summary of session concepts
+
+summary of commands / snippets that are useful for tutorials, challenges (for easy way of doing copy-paste)
+
+## Activities
+
+List of activities:
+
+tutorials (short description and objective, reference to activity folder in repository)
+
+challenges (short description and objective, reference to activity folder in repository)
+
+### Activity 1
+
+Description of activity, reference to activity folder, hints, public URL
+
+### Activity 2
+
+Description of activity, reference to activity folder, hints, public URL
+
+## Further Reading
```

Comment:
For me it makes more sense that the Further Reading chapter comes right after the Summary, before the Activities

Reply:
Makes sense. Will update.